### PR TITLE
gnome-session: update to 3.32.0

### PR DESCRIPTION
**Pull request by Oreo639**
~~This is also for resolving void-linux/void-packages#9714~~ My bad, it isn't necessary.

### CODE REVIEW

**Comment by Cogitri** [comment]
Works for me without this.

**Comment by Oreo639** [comment]
Ok. I could be mistaken, as I have not tried using the gnome-shell with the latest mutter without the latest gnome-session, only the latest mutter, and when I checked the logs, there were warnings coming from gnome-session-binary, which was why I assumed that this needed to be updated too.

**Comment by Cogitri** [comment]
Ah, no, gnome-session-binary most likely only proxied some other warnings.

**Comment by Oreo639** [comment]
Yeah. Do you want me to close this, or leave it open?

**Comment by Cogitri** [comment]
If you want to you can leave this open and update it once 3.32.0 has been released, or you close it and I'll do it. Just do it the way you like it the best :)

**Comment by Cogitri** [comment]
3.32.0 has been released.

**Comment by Oreo639** [comment]
Thanks for letting me know.

File: srcpkgs/gnome-session/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'gnome-session'
 pkgname=gnome-session
-version=3.30.1
+version=3.32.0
 revision=3
```
Comment (Cogitri) [inline srcpkgs/gnome-session/template]: Revision should be 1

**Comment by Oreo639** [comment]
Woops. Thx.

**Comment by Oreo639** [comment]
FIxed.

**Pull request merged**